addons/pos_gift_card/models/gift_card.py

```diff
@@ -22,7 +22,7 @@ def _compute_balance(self):
         super()._compute_balance()
         for record in self:
             confirmed_line = record.redeem_pos_order_line_ids.filtered(
-                lambda l: l.order_id.state == "paid"
+                lambda l: l.order_id.state in ('paid', 'done')
```

Comment:
I believe you should also include `'invoiced'` state here.

Reply:
Thanks for the review @caburj 
All things fixed! Could you check the update, please?